config/base.yml

```diff
@@ -7,6 +9,35 @@ Bundler/OrderedGems:
 Layout/ArgumentAlignment:
   EnforcedStyle: with_fixed_indentation
 
+Layout/ArrayAlignment:
+  Enabled: false
+
+Layout/ArrayAlignmentExtended:
+  Description: >-
+                 Align the elements of an array literal if they span more than
+                 one line.
+  # Alignment of elements of a multi-line array.
```

Comment:
@nikitasakov do we really need to add so many comments here?

Reply:
looks like all these comments should be in the file with ruby logic (can be wrong)

Reply:
@roman-dubrovsky I used original rubocop repo as an example and they write comments in these cops in config file - https://github.com/rubocop-hq/rubocop/blob/master/config/default.yml#L224

I can remove them if you don't like it.

Reply:
@nikitasakov can you answer me the question of why we should add or remove it?

Reply:
@roman-dubrovsky When I use rubocop (datarockets-style in our case) and want to change some cop (use another flag for example) I want to understand interface of this cop and how it will work with new flag.

So it is convenient for me to open config file and check it. And I shouldn't investigate files system of repo and understand where source code of this cop is situated.

Reply:
So I would like to have these comments in config

Reply:
@nikitasakov I think it does not really matter. First of all, nobody uses comments in yml file for investigating cop. Instead of it, I want to suggest you adds additional md file (like https://github.com/rubocop-hq/rubocop/blob/master/manual/cops_layout.md#layoutaccessmodifierindentation)

Here, I'm wondering if rubocop can use it for their CLI?

Reply:
I've checked CLI and couldn't find any flag that uses these comments. Will remove them